lib/plugins/aws/package/lib/getHashForFilePath.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const memoize = require('memoizee');
+const crypto = require('crypto');
+const fs = require('fs');
+
+const getHashForFilePath = memoize(async filePath => {
+  const fileHash = crypto.createHash('sha256');
+  fileHash.setEncoding('base64');
+  try {
+    fileHash.write(await fs.promises.readFile(filePath));
+    fileHash.end();
+    return fileHash.read();
+  } catch (error) {
+    throw new Error(
+      `Error: ${error} encountered during hash calculation for provided filePath: ${filePath}`
+    );
+  }
+});
```

Comment:
Let's ensure `promise: true` option. (`memoizee` was not updated in a while, and does not have automatic _async_ functions detection)

Reply:
good catch, I didn't encounter any error and wrongly assumed that it just works out of the box without issues :+1: